<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-size:16px;
	font-family: Arial;
	font-weight:normal;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size:25px;
	font-family:Arial;
}</string>
    </property>
    <property name="text">
     <string>Currency Converter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-style:italic;
}</string>
    </property>
    <property name="text">
     <string>*** The exchange rate is updated every day using the BNR official website</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>292</width>
      <height>308</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="convertFromLabel">
         <property name="styleSheet">
          <string notr="true">QLabel {
	margin-right:15px;
}</string>
         </property>
         <property name="text">
          <string>Convert from</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="convertFromComboBox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: #F0F0F0;
    border: 1px solid #8A8A8A;
    padding: 5px;
    border-radius: 5px;
    font-size: 14px;
    color: #333;
}

QComboBox:hover {
	border-color:gray;
	border-width:2px;
	font-weight:bold;
}

QComboBox::down-arrow {
	    image: url(&quot;F:/Python/Proiecte B/Currency Converter/down_arrow.png&quot;); /* Replace with the path to your custom arrow image */
		width:15px;
		height:20px;

}

QComboBox::drop-down {
    border-left: 1px solid #8A8A8A;
}



QComboBox QAbstractItemView {
    border:2px solid gray;
    background-color: #FFFFFF;
    selection-background-color: #3399FF;
    selection-color: #FFFFFF;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-bottom-left-radius: 3px;
	border-left-color:transparent;
}

QComboBox:on { 
    padding-left: 7px;
}

QComboBox:item {
	height:25px;
	padding-left:10px;
	background-color:red;
	
}

QComboBox::item:hover {
    background-color: #1ABC9C;
    color: #ECF0F1;
}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="toLabel">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="toComboBox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: #F0F0F0;
    border: 1px solid #8A8A8A;
    padding: 5px;
    border-radius: 5px;
    font-size: 14px;
    color: #333;
}

QComboBox:hover {
	border-color:gray;
	border-width:2px;
	font-weight:bold;
}

QComboBox::down-arrow {
	    image: url(&quot;F:/Python/Proiecte B/Currency Converter/down_arrow.png&quot;); /* Replace with the path to your custom arrow image */
		width:15px;
		height:20px;

}

QComboBox::drop-down {
    border-left: 1px solid #8A8A8A;
}



QComboBox QAbstractItemView {
    border:2px solid gray;
    background-color: #FFFFFF;
    selection-background-color: #3399FF;
    selection-color: #FFFFFF;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-bottom-left-radius: 3px;
	border-left-color:transparent;
}

QComboBox:on { 
    padding-left: 7px;
}

QComboBox:item {
	height:25px;
	padding-left:10px;
	background-color:red;
	
}

QComboBox::item:hover {
    background-color: #1ABC9C;
    color: #ECF0F1;
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ammountLabel">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ammountLineEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>27</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid gray;
    padding: 0 8px;
    selection-background-color: gray;
	border-top-right-radius:10px;
	min-height:25px;
	max-height:25px;
	font-size:14px;
}

QLineEdit:hover {
	border:2px solid black;
} 	

QLineEdit:focus {
	border:2px solid black;
} </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="convertButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>100</horstretch>
         <verstretch>50</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>199</width>
         <height>49</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	min-width:100px;
    max-width:100px;
    min-height:35px;
    max-height:35px;
	font-family: Arial;
	margin-left:95px;
	border-radius:10%;
	font-size:15px;
	border:2px solid black;
	font-weight:bold;
	color:black;
	margin-top:10px;
}

QPushButton:hover {
		background-color:black;
		color:white;
		font-weight:bold;
}

QPushButton:pressed {
	background-color:#232023;
	border-color:#232023;
}</string>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="styleSheet">
        <string notr="true">padding: 20px;
text-align:center;
min-width:250px;
max-width:250px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
